daemons/gptp/common/avbts_message.hpp

```diff
@@ -655,6 +688,72 @@ class FollowUpTLV {
 	int32_t getRateOffset() {
 		return cumulativeScaledRateOffset;
 	}
+
+    /**
+     * @brief  Updates the scaledLastGmFreqChanged private member
+     * @param  val Value to be set
+     * @return void
+     */
+    void setScaledLastGmFreqChange(int32_t val)
+    {
+        scaledLastGmFreqChange = PLAT_htonl(val);
+    }
+
+    /**
+     * @brief  Gets the current scaledLastGmFreqChanged value
+     * @return scaledLastGmFreqChange
+     */
+    int32_t getScaledLastGmFreqChange(void)
+    {
+        return scaledLastGmFreqChange;
+    }
+
+    /**
+     * @brief  Sets the gmTimeBaseIndicator private member
+     * @param  tbi Value to be set
+     * @return void
+     */
+    void setGMTimeBaseIndicator(uint16_t tbi)
+    {
+        gmTimeBaseIndicator = PLAT_htonl(tbi);
+    }
```

Comment:
You are right. I just kept the same format of the constructor of this method, which initializes it with "0" on the network order. 
since we are already doing it on network order, I think I should keep this method the way it is, and below I increment it on host order and then store it back on network order. 
Would that be acceptable for now ? We can open another issue to investigate storing this on network order vs host order so we dont forget about this as well.